components/docusign/actions/create-signature-request/common.mjs

```diff
@@ -1,4 +1,19 @@
 export default {
+  async additionalProps() {
```

Comment:
I don't see where these additional props are being used in the `run` function when sending the signature request.